# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1532979768 14400
#      Mon Jul 30 15:42:48 2018 -0400
# Branch stable
# Node ID 439e69ae7aeb759b011198663a538e6dbec96e1d
# Parent  96584a825b0fda01e01f4d0f9d32637907621d3d
bump version for 4.4.1-rc2

* configure.ac: Set version to 4.4.1-rc2 and release date to 2018-07-30.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -15,34 +15,34 @@ dnl Copyright (C) 1993-2018 John W. Eato
 ### GNU General Public License for more details.
 ###
 ### You should have received a copy of the GNU General Public License
 ### along with Octave; see the file COPYING.  If not, see
 ### <https://www.gnu.org/licenses/>.
 
 ### Initialize Autoconf
 AC_PREREQ([2.65])
-AC_INIT([GNU Octave], [4.4.1-rc1], [https://octave.org/bugs.html], [octave],
+AC_INIT([GNU Octave], [4.4.1-rc2], [https://octave.org/bugs.html], [octave],
         [https://www.gnu.org/software/octave/])
 
 ### Declare version numbers
 
 dnl Note that the version number is duplicated here and in AC_INIT because
 dnl AC_INIT requires it to be static, not computed from shell variables.
 
 OCTAVE_MAJOR_VERSION=4
 OCTAVE_MINOR_VERSION=4
-OCTAVE_PATCH_VERSION=1-rc1
+OCTAVE_PATCH_VERSION=1-rc2
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION argument.
 OCTAVE_VERSION="$PACKAGE_VERSION"
 
 OCTAVE_COPYRIGHT="Copyright (C) 2018 John W. Eaton and others."
 
-OCTAVE_RELEASE_DATE="2018-07-13"
+OCTAVE_RELEASE_DATE="2018-07-30"
 
 ## The "API version" is used as a way of checking that interfaces in the
 ## liboctave and libinterp libraries haven't changed in a backwardly
 ## incompatible way when loading .oct files.  A better way to do this is with
 ## library versioning, but not all systems support it.
 ##
 ## NOTE: This macro will be removed in a future version of Octave!
 ## If you insist on checking for features using a version number, use the
